src/main/java/frc/robot/Controls.java

```diff
@@ -133,6 +137,7 @@ private double getDriveRotate() {
     return input * MaxSpeed * inputScale;
   }
 
+  // all the controls binding for worlds
```

Comment:
What happens when we update the code? There are also some controls that weren't really used even though they did exist.